app-misc/fhem/fhem-5.8.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit user
+
+DESCRIPTION="A GPL'd perl server for house automation"
+HOMEPAGE="https://www.fhem.de/"
+SRC_URI="https://www.fhem.de/${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="doc"
+
+RDEPEND="dev-perl/Crypt-CBC
+	dev-perl/Device-SerialPort
+	dev-perl/Digest-CRC
+	dev-perl/JSON"
+
+DEPEND="media-gfx/pngcrush"
+
+RESTRICT="mirror"
```

Comment:
Why RESTRICT mirror?

Reply:
Good question. Looking at the Homepage and License, I don't see any reason, so I've removed it and pushed a new version.